index.js

```diff
@@ -1,18 +1,54 @@
-/***** Deliverable 1 *****/
+        /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "gold"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const img = document.querySelector("img")
+img.src = traveler.photo
+img.alt = traveler.name
 
+const h2 = document.querySelector("h2")
+h2.textContent = traveler.name
+
+const em = document.querySelector("em")
+em.textContent = traveler.nickname
 
 /***** Deliverable 4 *****/
 
 
+const ul = document.querySelector("#animals")
+traveler.animalSightings.forEach( function (sighting) {
+    
+let li = document.createElement("li")
+li.id = sighting.id
```

Comment:
Here, your goal is to use the `data-id` attribute, not the `id` -- they will show up differently in the HTML. Here's how you can use `data-id`:

```js
li.dataset.id = sighting.id
```

This will make the following element:

```html
<li data-id="3">
```